aip/client-libraries/4222.md

```diff
@@ -33,14 +33,21 @@ rpc CreateTopic(CreateTopicRequest) {
 In this case, the applicable variable is `parent`, and it refers to the
 `parent` field in `CreateTopicRequest`. When the user provides an instance of
 `CreateTopicRequest` to the method (or once the client library has built it, in
-the case of method overloads), the client library should then extract the key
-and value, URL-encode them, and append them to the `x-goog-request-params`
-header. Much like URL parameters, if there is more than one key-value pair, the
-`&` character is used as the separator.
+the case of method overloads), the client library must extract the key
+and value, and append them to the `x-goog-request-params` header. The key-value
+pairs **must** be URL-encoded per [RFC 6570][], section 3.2.2 [Simple String
```

Comment:
...be URL-encoded as described in RFC 6570 §3.2.2. Much like...

And linkify the whole thing as a single link, to the particular section.

Reply:
Done.